<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>675</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>675</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Graphique</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ressource_projet tableur/docs_projet/bar-chart.svg</normaloff>../ressource_projet tableur/docs_projet/bar-chart.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1061</width>
      <height>621</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>17</number>
       </property>
       <item row="9" column="0">
        <widget class="QLabel" name="couleurDeFondLabel">
         <property name="text">
          <string>Couleur de fond</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="Buttoncolor2">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="Buttoncolor3">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="couleur2">
         <property name="text">
          <string>Couleur par défaut 2</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="hauteurLabel">
         <property name="text">
          <string>Hauteur</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="largeurLabel">
         <property name="text">
          <string>Largeur</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="Spinmargex">
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="Combotype">
         <item>
          <property name="text">
           <string>Lignes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Points</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Barres</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="Spinhauteur">
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="value">
          <number>600</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="couleur3">
         <property name="text">
          <string>Couleur par défaut 3</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="Spinmargey">
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="Buttoncolor1">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="LinkVisited">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>138</red>
               <green>226</green>
               <blue>52</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="LinkVisited">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>138</red>
               <green>226</green>
               <blue>52</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="LinkVisited">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>138</red>
               <green>226</green>
               <blue>52</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="shortcut">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="margeYLabel">
         <property name="text">
          <string>Marge Y</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="typeLabel">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Pas Y</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="couleur1">
         <property name="text">
          <string>Couleur par défaut 1</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="couleurDesAxesLabel">
         <property name="text">
          <string>Couleur des axes</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="policeLabel">
         <property name="text">
          <string>Police</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="Spinlargeur">
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="value">
          <number>800</number>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="pasXLabel">
         <property name="text">
          <string>Pas X</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="margeXLabel">
         <property name="text">
          <string>Marge X</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QPushButton" name="Buttoncoloraxe">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QSpinBox" name="Spinpasx">
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QSpinBox" name="Spinpasy">
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QCheckBox" name="checkBoxgrill">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="Buttonfont">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QWidget" name="couleurDeFondWidget" native="true">
         <widget class="QPushButton" name="Buttoncolorfond">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>30</width>
            <height>27</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Montrer la grille</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1094</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionCharger"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuGraphique">
    <property name="title">
     <string>Graphique</string>
    </property>
    <addaction name="actionDessiner_le_graphique"/>
    <addaction name="actionSauvergarder_le_graphique_sous"/>
   </widget>
   <widget class="QMenu" name="menuhelper">
    <property name="title">
     <string/>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="action_propos_de_Qt"/>
    <addaction name="action_propos_de_Graphiques"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuGraphique"/>
   <addaction name="menuhelper"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCharger">
   <property name="text">
    <string>Charger</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionDessiner_le_graphique">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Dessiner le graphique </string>
   </property>
  </action>
  <action name="actionSauvergarder_le_graphique_sous">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sauvergarder le graphique sous ...</string>
   </property>
  </action>
  <action name="action_propos_de_Qt">
   <property name="text">
    <string>À propos de Qt</string>
   </property>
  </action>
  <action name="action_propos_de_Graphiques">
   <property name="text">
    <string>À propos de Graphiques</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
